pkgs/development/ocaml-modules/angstrom/default.nix

```diff
@@ -1,37 +1,25 @@
-{ stdenv, fetchFromGitHub, ocaml, findlib, dune, alcotest, result
-, bigstringaf
-}:
+{ stdenv, ocamlPackages }:
 
-if !stdenv.lib.versionAtLeast ocaml.version "4.03"
-then throw "angstrom is not available for OCaml ${ocaml.version}"
-else
-
-stdenv.mkDerivation rec {
+with ocamlPackages;
+dunePackage {
+  pname = "angstrom";
   version = "0.10.0";
-  name = "ocaml${ocaml.version}-angstrom-${version}";
 
-  src = fetchFromGitHub {
-    owner  = "inhabitedtype";
-    repo   = "angstrom";
-    rev    = "${version}";
-    sha256 = "0lh6024yf9ds0nh9i93r9m6p5psi8nvrqxl5x7jwl13zb0r9xfpw";
+  minimumOCamlVersion = "4.03";
+
+  src = {
+    github-owner = "inhabitedtype";
+    sha256 = "0l6gr7hqjmfbhc0y34h6v9sm1qq9zbba9w5ccmk9qbca7r488cyp";
```

Comment:
The hash should not change. But as Jörg mentioned above, the `src` attribute should be kept as is.